scenes/game/local_game.gd

```diff
@@ -3341,7 +3341,7 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			opposing_player.cannot_move = false
 		"bonus_action":
 			active_boost.action_after_boost = true
-		"boost":
+		"boost_additional":
 			var valid_zones = ['hand']
```

Comment:
can you just assert(active_boost) here?
These boost functions are all very fragile and you'll see in my Platinum branch I'm adding lots of comments and stuff.